src/evm/protocol/uniswap_v4/state.rs

```diff
@@ -51,6 +51,19 @@ pub struct UniswapV4State {
     pub hook: Option<Box<dyn HookHandler>>,
 }
 
+impl fmt::Debug for UniswapV4State {
```

Comment:
What was the purpose of this? Why wasn't the original Debug derivation good enough? It's probably worth to document that in the commit message because I'll likely be confused in the future

Reply:
aaahh no no, it was because I didn't want to log everything. For example all the storage slots for the hooks. I only implemented this for protocols that would have a lot of "spammy" info in their attributes (adding this to the PR description)